lib/Conversion/TorchToLinalg/Pooling.cpp

```diff
@@ -366,6 +473,86 @@ class ConvertAtenMaxPool2dWithIndicesOp
 };
 } // namespace
 
+
+namespace {
+class ConvertAtenAvgPool1dOp : public OpConversionPattern<AtenAvgPool1dOp> {
```

Comment:
This pattern shares a lot of code with `ConvertAtenAvgPool2dOp`. Common logic should be identified and refactored to avoid repetition.

Reply:
Fixed,  have already merge these two functions.